pg_dump.go

```diff
@@ -52,7 +52,11 @@ func (x *Dump) Exec(opts ExecOptions) Result {
 	cmd.Env = append(os.Environ(), x.EnvPassword)
 	stderrIn, _ := cmd.StderrPipe()
 	go func() {
```

Comment:
I think we can move this code into a new function so we don't have to write it twice 👍

Reply:
Something like this?

	go func() {
		streamOutput(stderrIn, opts, result)
	}()

And then

	func streamOutput(stderrIn io.ReadCloser, opts ExecOptions, result Result) {
		output, err := streamExecOutput(stderrIn, opts)
		if err != nil {
			result.Error = &ResultError{Err: err, CmdOutput: output}
		}
		result.Output = output
	}

If yes, where would you place the function?

Reply:
Good point! :smile: Should be done now